src/app/app-routing.module.ts

```diff
@@ -0,0 +1,38 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { SearchResultsComponent } from './youtube/pages/search-results/search-results.component';
+import { NotFoundComponent } from './core/components/not-found/not-found.component';
+import { AuthGuard } from './shared/guards/auth.guard';
+import { LoginComponent } from './auth/components/login/login.component';
+import { RegistrationComponent } from './auth/components/registration/registration.component';
+
+const routes: Routes = [
+  {
+    path: 'login',
+    component: LoginComponent,
+    loadChildren: () => import('./auth/auth.module').then(m => m.AuthModule),
+  },
+  {
+    path: 'registration',
+    component: RegistrationComponent,
+    loadChildren: () => import('./auth/auth.module').then(m => m.AuthModule),
+  },
```

Comment:
it is strange to see to different lazy routes to have the same module. I guess we should or split Authnodule into to different parts for login and registration, or combine both routes into one